tdrs-frontend/nginx/mime.types

```diff
@@ -1,10 +1,12 @@
 types {
   text/html html htm shtml;
   text/css css;
+  text/css chunk.css;
```

Comment:
This is really not a requirement, but I thought it is better to add to save us from future problem.